admin/muninlite/Makefile

```diff
@@ -24,7 +24,7 @@ define Package/muninlite
   CATEGORY:=Administration
   DEPENDS:=+xinetd
   TITLE:=Munin node implemented in shell
-  URL:=http://sourceforge.net/projects/muninlite/
+  URL:=https://github.com/munin-monitoring/muninlite
   PKG_MAINTAINER:=Jonathan McCrohan <jmccrohan@gmail.com>
```

Comment:
Wrong variable and wrong place.

Should be before PKG_LICENSE.

Reply:
Thanks, done.